projects/mysadna2/src/app/app-routing.module.ts

```diff
@@ -8,9 +8,9 @@ import { ProjectDetailsComponent } from './project-details/project-details.compo
 
 const routes: Routes = [
   {path: 'users', component: UsersComponent},
-  {path: 'user-details', component: UserDetailsComponent},
+    {path: 'user-details/:userId', component: UserDetailsComponent},
```

Comment:
I was actually following an example from the angular tutorial, where they added a space like that.
I assumed it was because the '...-details' was not a standalone component.

Reply:
It makes sense actually!
Problem is we need to add custom format rules to avoid it by editor formatter.